contracts/vaults/Claim.sol

```diff
@@ -46,20 +46,30 @@ contract Claim is Base {
             bountyPercentage: _bountyPercentage,
             committee: msg.sender,
             // solhint-disable-next-line not-rely-on-time
-            createdAt: block.timestamp
+            createdAt: block.timestamp,
+            isChallenged: false
         });
         emit SubmitClaim(_pid, msg.sender, _beneficiary, _bountyPercentage, _descriptionHash);
     }
 
+    function isChallenged(uint256 _claimId) onlyArbitrator external {
```

Comment:
rename to challengeClaim?
and also why is this called by the arbitrator? we said anyone can challenge

Reply:
anyone can challenge, but they will ahve to to it via the arbitrator contract (which will ask them for a deposit so we do not get spammed)